components/raftstore-v2/src/operation/ready/mod.rs

```diff
@@ -395,6 +411,71 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
     pub fn on_wait_flush(&mut self, ch: crate::router::FlushChannel) {
         self.async_writer.subscirbe_flush(ch);
     }
+
+    pub fn on_role_changed<T>(&mut self, ctx: &mut StoreContext<EK, ER, T>, ready: &Ready) {
```

Comment:
Could you add a comment if there's any behavior change between v1 and v2.

Reply:
There is no difference. Missing part is to be implemented.